packages/terra-table/tests/jest/ColumnResizeHandle.test.jsx

```diff
@@ -15,6 +15,11 @@ afterAll(() => {
 });
 
 describe('ColumnResizeHandle', () => {
+  const mockSetIsActive = jest.fn();
```

Comment:
Why is this not just defined in the tests that require it?  This does not seem like something we should have to validate every test.